File: devcamp2018/app/src/main/java/com/psato/devcamp/presentation/MVVM/QueryActivity.java
package com.psato.devcamp.presentation.MVVM;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.psato.devcamp.R;

public class QueryActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        DataBindingUtil.setContentView(this, R.layout.activity_list);
        if (savedInstanceState == null) {
            getSupportFragmentManager()
                    .beginTransaction()
                    .add(R.id.fragment_content, new com.psato.devcamp.presentation.MVVM.QueryFragment())
                    .commit();
        }
    }
}

File: devcamp2018/app/src/main/java/com/psato/devcamp/presentation/search/QueryActivity.kt
package com.psato.devcamp.presentation.search

import android.databinding.DataBindingUtil
import android.databinding.ViewDataBinding
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.psato.devcamp.R

class QueryActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        DataBindingUtil.setContentView<ViewDataBinding>(this, R.layout.activity_list)
        if (savedInstanceState == null) {
            supportFragmentManager
                    .beginTransaction()
                    .add(R.id.fragment_content, com.psato.devcamp.presentation.search.QueryFragment())
                    .commit()
        }
    }
}
